stdlib/CMakeLists.txt

```diff
@@ -93,15 +93,11 @@ add_custom_target(swift-test-stdlib ALL
 
 if(SWIFT_STDLIB_ENABLE_RESILIENCE)
   set(STDLIB_SIL_SERIALIZE_ALL)
```

Comment:
You can remove all mentions of STDLIB_SIL_SERIALIZE_ALL from the CMake scripts now, and also remove the flag from build-script-impl